l-homme.csl

```diff
@@ -164,41 +182,47 @@
               </group>
             </group>
           </if>
-          <else-if type="webpage">
-            <text macro="title" font-style="italic" suffix=". "/>
+          <else-if type="webpage post post-weblog" match="any">
+            <text macro="title" font-style="italic"/>
             <group>
               <text variable="URL" prefix=", "/>
               <text prefix=" (" macro="access" suffix=")"/>
             </group>
           </else-if>
           <else-if type="article-journal article-magazine article-newspaper broadcast personal_communication thesis entry-dictionary entry-encyclopedia" match="any">
-            <text macro="title" quotes="true" suffix=". "/>
             <group delimiter=", ">
-              <text variable="container-title" font-style="italic"/>
-              <text variable="volume" prefix="vol.&#160;"/>
-              <text variable="issue" prefix="n°&#160;"/>
+              <text macro="title" quotes="true"/>
+              <group delimiter=": ">
```

Comment:
Why not do something like this to get what they want, or am I misunderstanding the problem?
```suggestion
              <group delimiter="&#160;:&#160;">
```

Reply:
but the style is set to `fr` anyway?

Reply:
I've added an english style. Let me know if the naming is ok.